deps/rabbit/src/rabbit_deprecated_feature_extra.erl

```diff
@@ -0,0 +1,62 @@
+%% This Source Code Form is subject to the terms of the Mozilla Public
+%% License, v. 2.0. If a copy of the MPL was not distributed with this
+%% file, You can obtain one at https://mozilla.org/MPL/2.0/.
+%%
+%% @copyright 2018-2023 VMware, Inc. or its affiliates.
+%%
+%% @doc
+%% This module provides extra functions unused by the feature flags
+%% subsystem core functionality.
+
+-module(rabbit_deprecated_feature_extra).
+
+-export([cli_info/1]).
+
+-type cli_info() :: [cli_info_entry()].
+%% A list of deprecated feature properties, formatted for the RabbitMQ CLI.
+
+-type cli_info_entry() :: [{name, rabbit_feature_flags:feature_name()} |
+                           {deprecation_phase, rabbit_deprecated_features:deprecation_phase()} |
+                           {provided_by, atom()} |
+                           {desc, string()} |
+                           {doc_url, string()}].
```

Comment:
Is it possible to use a map? A proplist is kind of obsolete nowadays and unpracticle in every ways.

Also please stay in the 80 column boundary.

Reply:
A map is not a problem, it just follows `rabbit_ff_extra:cli_info/1` format.